docs/remove_files.py

```diff
@@ -0,0 +1,19 @@
+import sys
+from pathlib import Path
+
+from jax.typing import ArrayLike
+
+
+def remove_files(pattern: ArrayLike):
```

Comment:
```suggestion
def remove_files(pattern: str):
```
And this should be a string I guess?